api/RSSFeed.php

```diff
@@ -184,13 +184,17 @@ public function Description() {
 	}
 
 	/**
-	 * Output the feed to the browser
+	 * Output the feed to the browser.
+	 *
+	 * @param	SS_HTTPResponse|null	$response
+	 * @return	HTMLText
 	 */
-	public function outputToBrowser() {
+	public function outputToBrowser(SS_HTTPResponse $response = null) {
```

Comment:
Ah, this is not a compatible change I'm afraid... this can't even go into 3.x if this is the change that's necessary. It'll have to go into master.